action.yml

```diff
@@ -97,6 +97,14 @@ runs:
         if [[ "${{ inputs.phpunit_fail_on_warning }}" == "true" ]]; then
           PHPUNIT_OPTIONS="$PHPUNIT_OPTIONS --fail-on-warning"
         fi
+        # Special filtering for silverstripe/framework testsuites
+        if [[ "$PHPUNIT_SUITE" == "framework-orm" ]]; then
+          PHPUNIT_OPTIONS="$PHPUNIT_OPTIONS --filter /ORM/"
```

Comment:
Based on my testing locally, this needs to be `/ORM/` specifically and not `tests/php/ORM` as the `<directory>` in the `<testsuite>` contains `tests/php`